.github/workflows/vsenv.bat

```diff
@@ -0,0 +1,17 @@
+@echo off
+
+SET VSWHERE="C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere"
```

Comment:
C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere, is this folder exist for everyone?